github/MainClass.py

```diff
@@ -341,6 +343,22 @@ def get_organizations(self, since=github.GithubObject.NotSet):
             url_parameters,
         )
 
+    def get_enterprise(self, login):
+        """
+        :calls: `GET /enterprise/{enterprise} <https://docs.github.com/en/enterprise-cloud@latest/rest/enterprise-admin>`_
+        :param login: string
+        :rtype: :class:`Enterprise`
+        """
+        assert isinstance(login, str), login
+        firstURL = f"/enterprises/{login}"
+        # There is no native "/enterprise/{enterprise}" api, so this function is a hub for under apis of enterprise administration.
+        headers = {}
+        data = {}
+        # The response doesn't have the key of login and url, manually add it to data.
+        data["login"] = login
+        data["url"] = self.__requester.base_url + firstURL
+        return github.Enterprise.Enterprise(self.__requester, headers, data, completed=True)
```

Comment:
This then simplifies to
```suggestion
        assert isinstance(login, str), login
        # There is no native "/enterprise/{enterprise}" api, so this function is a hub for apis that start with "/enterprise/{enterprise}".
        return github.Enterprise.Enterprise(self.__requester, login)
```

Reply:
Thank you. Fixed it.